templates/basic/README.md

```diff
@@ -0,0 +1,40 @@
+# Welcome to React Router!
+
+- 📖 [React Router docs](https://reactrouter.com/dev)
```

Comment:
Right now I have everything pointing to `/dev` since this is all in the pre-release. We'll of course need to clean this up before the final release

Reply:
That link doesn't work, https://reactrouter.com/en/dev that one does it

Reply:
Thanks Sergio! Yeah, the links are anticipating some changes to the react-router site. I'm planning to go through them all again before we release the prerelease